.github/waitfor.sh

```diff
@@ -0,0 +1,34 @@
+# Wait for something in 'juju status' to reach a given state
```

Comment:
Since this is specifically for status, should it be called `wjatforstatus.sh` ?
Or better keep the filename and make waitforstatus a func and then we can add other wait fors as well?
The actions that need it could just source it call the method(s)?

Reply:
all reasonable ideas, I think it's fine to leave it for now, and let's wait to see what we need in the future. It's easy enough to make any of those changes down the road.